tests/unit/bokeh/core/property/test_visual.py

```diff
@@ -273,6 +274,19 @@ def test_transform_data_url(self) -> None:
         image = "data:image/png;base64,"
         assert prop.transform(image) == image
 
+    def test_transform_path(self) -> None:
+        with tempfile.NamedTemporaryFile() as file:
+            path = Path(file.name)
+            image = PIL.Image.new('RGBA', size=(50, 50), color=(155, 0, 0))
+            image.save(file, 'png')
+            prop = bcpv.Image()
+            assert prop.transform(path).startswith("data:image/png")
```

Comment:
@mattpap unfortunately this is failing on windows, possible due to
>  Whether the name can be used to open the file a second time, while the named temporary file is still open, varies across platforms (it can be so used on Unix; it cannot on Windows). 

(i.e. maybe `image.save` attempts to re-open?)

Reply:
I resolved this with a bit of trickery (using a temporary directory instead).